src/main/kotlin/me/kject/internal/KJectImpl.kt

```diff
@@ -36,10 +42,14 @@ internal object KJectImpl {
             if (!disposed || _scope != null || _context != null) throw AlreadyInitializeException()
             disposed = false
 
-            _scope = scope
+            _scope = CoroutineScope(
+                EmptyCoroutineContext +
+                        Job(scope.coroutineContext.job) +
+                        CoroutineName("KJect")
```

Comment:
Is the extra indent on purpose?

Reply:
The indent is the default formating in kotlin.
I think that comes from [Wrap Chained Calls](https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls), as bassicly the `+` is just a call to the function `plus`.